setup.py

```diff
@@ -1,8 +1,43 @@
 try:
-    from setuptools import setup
+    from setuptools import setup, Command
 except ImportError:
     from distutils.core import setup
 
+
+class LiveTestCommand(Command):
+
+    description = "Run tests against a live server"
+
+    # [('long', 'short', 'description')]
+    user_options = [('server=', 's', "Server to connect to"),
+                    ('site=', 't', "Sute to connect to"),
+                    ('username=', 'u', "Username for tests (system admin)")]
+
+    def initialize_options(self):
+        self.server = None
+        self.username = None
+        self.password = None
+        self.site = None
+
+    def finalize_options(self):
+        if self.server is None:
+            self.server = "DEFAULT SERVER"
+        if self.username is None:
+            self.username = "DEFAULT USERNAME"
+        if self.site is None:
+            self.site = ""
+
+        import getpass
+        self.password = getpass.getpass()
+
+    def run(self):
+        import subprocess as sp
+        import os
+
+        with open(os.devnull, 'wb') as silent:
+            sp.check_call(['python', 'smoke.py', self.server, self.username, self.password, self.site], stdout=silent)
```

Comment:
Rather than do it this way, could we do a directory of smoke tests and then run it with `python -m unittest smoke`?  This would mean we wouldn't need to write our own test runner just to run the smoke tests, then we can have a Base Smoke Test class that reads the arguments from the command line, and all tests derive from that test class.